wait-for-it/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM alpine:3.10
```

Comment:
Looks like this is a few versions behind also

Reply:
Updated